client/src/styles/PortfolioTable.scss

```diff
@@ -105,15 +111,21 @@ $table-border-dark: 1px solid $dark;
         position: sticky;
         top: 0;
         z-index: 50;
+        white-space: nowrap;
 
         tr {
           height: 30px;
+          div {
+            font-weight: 600 !important;
```

Comment:
is this just taking the place of `font-weight: bold` that was removed from `$gray-dark`?

Reply:
Yeah, it was suggested to use the number rather than names, but then I also realized that there was some default styling (I think coming from the spectre.css we use, or maybe just general defaults) that was making it bold, and adding the specificity by included the nested div that are in each cell helped override that)

Reply:
oh cool, that's really good to know for future reference!